Mathlib/GroupTheory/SpecificGroups/Cyclic.lean

```diff
@@ -40,6 +40,7 @@ For the concrete cyclic group of order `n`, see `Data.ZMod.Basic`.
 cyclic group
 -/
 
+assert_not_exists TwoSidedIdeal
```

Comment:
No objection to this PR, but I hope we can eventually add `Module` and even `Ring` here. If you want to add them as a TODO I'd be happy. :-) But no worries.

Reply:
I'll play around with it!